test/unit/Home.test.js

```diff
@@ -1,57 +1,62 @@
 /* eslint-env mocha */
 import React from 'react';
 import { expect } from 'chai';
-import { shallow, mount } from 'enzyme';
+import { mount } from 'enzyme';
 
 import Home from '../../src/app/components/Home/Home.jsx';
 
 describe('Home', () => {
   let component;
 
   before(() => {
-    component = shallow(<Home />);
+    component = mount(<Home isLoading={false} />);
   });
 
   it('should be wrapped in a .home class', () => {
-    expect(component.find('.home')).to.exist;
+    expect(component.find('.home').length).to.equal(1);
     expect(component.find('div').first().hasClass('home')).to.equal(true);
   });
 
-  it('should render a .page-header div and page title', () => {
-    const pageHeader = component.find('.page-header');
+  it('should render a hero-type banner page title', () => {
+    const pageHeader = component.find('.nypl-homepage-hero');
 
-    expect(component.find('.page-header')).to.have.length(1);
+    expect(pageHeader).to.have.length(1);
+    expect(pageHeader.find('h1')).to.have.length(1);
+    expect(pageHeader.find('h1').text()).to.equal('Shared Collection Catalog');
+    expect(pageHeader.contains(<h1>Shared Collection Catalog</h1>)).to.equal(true);
+  });
 
-    expect(pageHeader.find('h2')).to.have.length(1);
-    expect(pageHeader.find('h2').text()).to.equal('New York Public Library Research Catalog');
+  it('should contains a Search component in the banner', () => {
+    const pageHeader = component.find('.nypl-homepage-hero');
+    expect(pageHeader.find('Search')).to.have.length(1);
+  });
 
-    expect(pageHeader.contains(<h2>New York Public Library Research Catalog</h2>)).to.equal(true);
+  it('should contain an h2', () => {
+    const h2 = component.find('h2');
+    expect(h2.length).to.equal(1);
+    expect(h2.text()).to.equal('Research at NYPL');
   });
 
-  it('should contains a Breadcrumbs component', () => {
-    const pageHeader = component.find('.page-header');
+  it('should contain five images', () => {
+    const imageBlocks = component.find('img.nypl-quarter-image');
 
-    expect(pageHeader.find('Breadcrumbs')).to.have.length(1);
+    expect(component.find('img').length).to.equal(5);
+    expect(imageBlocks.length).to.equal(5);
   });
 
-  it('should contains a Search component', () => {
-    expect(component.find('Search')).to.have.length(1);
-    // expect(component.contains(
```

Comment:
the test won't pass here?

Reply:
What do you mean?

Reply:
Because you commented out the test task here but removed it, I assume we will need it later but the test task won't pass now?

Reply:
It was commented out before because I was still trying to test it. But, now I removed it because there's no search component on the homepage.